include/net/golioth.h

```diff
@@ -295,6 +295,21 @@ int golioth_lightdb_set(struct golioth_client *client, const uint8_t *path,
 			enum coap_content_format format,
 			uint8_t *data, uint16_t data_len);
 
+/**
+ * @brief Delete value in Golioth's Light DB
+ *
+ * Delete value in Light DB.
+ *
+ * @param client Client instance
+ * @param path Light DB resource path
+ * @param format Format of payload
+ *
+ * @retval 0 On success
+ * @retval <0 On failure
+ */
+int golioth_lightdb_delete(struct golioth_client *client, const uint8_t *path,
+			enum coap_content_format format);
```

Comment:
`format` should not be needed, as there is no payload actually sent.

Reply:
Ah, I thought that might be tied to the path specification in some way.  I'll test without.